classes/form/section_select_form.php

```diff
@@ -87,6 +88,24 @@ public function definition() {
             $radioarray[] = $mform->createElement('radio', 'targetsectionnum',
                 '', $sectionname, $sectionnum, ['class' => 'mt-2']);
         }
+
+        $course = $DB->get_record('course', array('id' => $targetcourseid), '*', MUST_EXIST);
```

Comment:
Well, the course should definitely exist here. Otherwise `get_fast_modinfo` would fail. So we do not really need to query DB here again, right?

`$targetcoursemodinfo->get_course()` should be fine here?

Reply:
yep